.github/workflows/e2e-tests.yml

```diff
@@ -185,21 +185,16 @@ jobs:
         run: tar -xf frontend-e2e-build.tar.zst
 
       - name: Run server
-        working-directory: frontend/
-        run: (screen -dmS server -L yarn start) && sleep 3 && (tail -f screenlog.0 &)
+        run: |
+          yarn start --filter=frontend &
+          sleep 10
+          curl -L --max-time 120 http://127.0.0.1:3001/ || exit 1
```

Comment:
Redirect the output to /dev/null otherwise the CI log is spammed with HTML output